wsclient/src/test/java/org/hpccsystems/ws/client/platform/test/PlatformTester.java

```diff
@@ -203,7 +204,31 @@ else if (currentParam.matches(WSSQLPORTPATTERN))
             platform.checkInHPCCWsClient(client2);
             platform.checkInHPCCWsClient(client3);
             platform.checkInHPCCWsClient(client4);
-
+            
+            DropZone[] dropzones = platform.getDropZones();
+            for(int i = 0; i < dropzones.length; i++)
+            {
+            	System.out.println("Dropzone Name: " + dropzones[i].getName());
+            	String[] cna = dropzones[i].getConfNetAddresses();
+            	String[] na = dropzones[i].getNetAddresses();
+            	if(cna != null && na != null)
+            	{
+            		if(cna.length != na.length)
+            		{
+            			System.out.println("confNetAddress list does not match addressList length");
+            		}
+            		else
+            		{
+            			for(int b = 0; b < cna.length; b++)
+            			{
+            				System.out.println("\tconfNetAddress: " + cna[b]);
+            				System.out.println("\tnetAddress:     " + na[b]);
+            				System.out.println("\t-------------------------------------");
+            			}
+            		}
+            	}
+            }
+            
```

Comment:
can we also add a test using HPCCFileSprayClient.listFiles? HPCCFileSprayClient.fetchLocalDropZones, HPCCFileSprayClient.fetchDropZones(string), etc to make sure that functionality continues to work w/ the new DZ structures?